packages/blockchain-common/src/registries/FabricEnvironmentRegistryEntry.ts

```diff
@@ -18,20 +18,21 @@ export enum EnvironmentFlags {
     ENVIRONMENT = 1 << 0,
     ANSIBLE = 1 << 1,
     MANAGED = 1 << 2,
-    LOCAL = 1 << 3,
+    LOCAL = 1 << 3, // Old
     OPS_TOOLS = 1 << 4,
     SAAS = 1 << 5,
-    MICROFAB = 1 << 6
+    LOCAL_MICROFAB = 1 << 6,
+    MICROFAB = 1 << 7
 }
 
 export enum EnvironmentType {
     ENVIRONMENT = EnvironmentFlags.ENVIRONMENT,
     ANSIBLE_ENVIRONMENT = EnvironmentFlags.ANSIBLE,
-    MANAGED_ANSIBLE_ENVIRONMENT = EnvironmentFlags.MANAGED | EnvironmentFlags.ANSIBLE,
     LOCAL_ENVIRONMENT = EnvironmentFlags.LOCAL | EnvironmentFlags.MANAGED | EnvironmentFlags.ANSIBLE, // Local environments,
     OPS_TOOLS_ENVIRONMENT = EnvironmentFlags.OPS_TOOLS,
     SAAS_OPS_TOOLS_ENVIRONMENT = EnvironmentFlags.OPS_TOOLS | EnvironmentFlags.SAAS,
-    MICROFAB_ENVIRONMENT = EnvironmentFlags.MICROFAB
+    LOCAL_MICROFAB_ENVIRONMENT = EnvironmentFlags.LOCAL_MICROFAB,
+    MICROFAB_ENVIRONMENT = EnvironmentFlags.MICROFAB | EnvironmentFlags.LOCAL_MICROFAB
```

Comment:
Shouldn't we just leave the old flags as they are, and create a new environment type with the local and the microfab flags?

LOCAL_MICROFAB_ENVIRONMENT = EnvironmentFlags.LOCA | EnvironmentFlags.MICROFAB